docs/_includes/api/overview.html

```diff
@@ -1,6 +1,7 @@
 {% include anchor.html edit="true" title="API overview" hash="overview" %}
 
```

Comment:
The API overview is where I talk about the differences between callbacks, promises, and async functions. Note that I broke it up into separate subsections.